### PR TITLE
PA-1033 Integrate PIMS API with Elastic Search and Kibana Logging

### PR DESCRIPTION
**Pull request by goemen**

### CODE REVIEW

**Review by jdtoombs — APPROVED** [review]

File: backend/api/appsettings.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
     }
   },
   "ElasticConfiguration": {
```
Comment (Fosol) [inline backend/api/appsettings.json:25]: Can you rename to `ElasticSearch`

File: openshift/templates/logging/elasticsearch-template.yml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      namespace: ${PROJECT_NAME}
+      name: ${APP_NAME}
+      labels:
+        app: ${APP_NAME}
+        role: ${ROLE_NAME}
+    data:
+      elasticsearch.yml: |-
+        xpack.security.enabled: true
+  - apiVersion: v1
+    kind: ConfigMap
```
Comment (Fosol) [inline openshift/templates/logging/elasticsearch-template.yml:22]: This should be a secret, not a configmap

Comment (Fosol) [reply openshift/templates/logging/elasticsearch-template.yml:22]: You should probably also include the username as a configuration item

File: openshift/templates/logging/kibana-template.yml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+kind: Template
+metadata:
+  name: pims-kibana-deploy
+  annotations:
+    openshift.io/display-name: PIMS Kibana
+    description: Deployment template for Kibana
+    tags: pims-logging
+objects:
+  - apiVersion: v1
+    kind: ConfigMap
```
Comment (Fosol) [inline openshift/templates/logging/kibana-template.yml:11]: Don't include the secrets in the configmap

**Review by Fosol — COMMENTED** [review]
Should make some changes to the secrets

**Review by devinleighsmith — APPROVED** [review]

**Review by Fosol — APPROVED** [review]

**Pull request merged**